### PR TITLE
chore(circleci): add integration env vars to circle builds

### PR DESCRIPTION
**Pull request by adamweeks**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -66,6 +66,14 @@ jobs:
     <<: *job_common
     environment:
       - NODE_ENV: "test"
+      - IDBROKER_BASE_URL: "https://idbrokerbts.webex.com"
```
Comment (mattnorris) [inline .circleci/config.yml]: Are WHISTLER and HYDRA needed?

File: .circleci/config.yml [hunk 2]
```diff
@@ -66,6 +66,14 @@ jobs:
...
+      - IDENTITY_BASE_URL: "https://identitybts.webex.com"
+      - CONVERSATION_SERVICE: "https://conversation-intb.ciscospark.com/conversation/api/v1"
+      - ENCRYPTION_SERVICE_URL: "https://encryption-intb.ciscospark.com/encryption/api/v1"
+      - ACL_SERVICE_URL: "https://acl-intb.ciscospark.com/acl/api/v1"
+      - ATLAS_SERVICE_URL: "https://atlas-intb.ciscospark.com/admin/api/v1"
+      - WDM_SERVICE_URL: "https://wdm-intb.ciscospark.com/wdm/api/v1"
```
Comment (mattnorris) [inline .circleci/config.yml]: Can you alphabetize so it's easier to compare across codebases?

File: test/journeys/specs/smoke/demo.js [hunk 3]
```diff
@@ -25,10 +25,15 @@ describe('demo widget', () => {
...
+    // Refresh the browsers so an input timeout shouldn't happen
+    browser.refresh();
   });
 
-  it('saves token for users', () => {
+  it('saves token for local user', () => {
     saveToken(browserLocal, mccoy.token.access_token);
+  });
+
+  it('saves token for remote user', () => {
```
Comment (mattnorris) [inline test/journeys/specs/smoke/demo.js]: 👍

File: .circleci/config.yml (hunk 1 above)
Comment (adamweeks) [reply .circleci/config.yml]: We aren't testing anything with whistler yet. I'll add hydra, although tests are passing without it somehow

**Pull request merged**